### ISSUE TITLE
"info.length_seconds will be removed in a future release" warning message

### ISSUE THREAD

**Issue by SugarD-x** [Asshat-Gaming/DummyBot#7]
Due to an outdated reference in the  ytdl-core-discord dependency, (which must be fixed on their end), the following warning message appears when playing music:
```
`info.length_seconds` will be removed in a near future release, use `info.videoDetails.lengthSeconds` instead.
```
This has been reported to ytdl-core-discord, and the issue can be tracked here: https://github.com/amishshah/ytdl-core-discord/issues/244

**Comment by EasyXploit** [comment]
As described in [#244](https://github.com/amishshah/ytdl-core-discord/issues/244) this issue has been fixed in [v1.2.3](https://www.npmjs.com/package/ytdl-core-discord/v/1.2.3).
This issue needs to be closed aswell.

**Comment by SugarD-x** [comment]
The dependency has been updated to resolve this.
